eta/core/annotations.py

```diff
@@ -1,11 +1,7 @@
 '''
 Core utilities for rendering annotations on media.
 
-@todo support for rendering video-level attributes
```

Comment:
@yashbhalgat note that I went ahead and supported the other two todo items

Reply:
Yes.. This looks good now